### PR TITLE
Add ScottoSlant Keyboard

### PR DESCRIPTION
**Pull request by joe-scotto**
Add ScottoSlant handwired keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/scottokeebs/scottoslant/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Compiling
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/jscotto/scottoslant:default
+
+Flashing example for this keyboard:
+
+    make handwired/jscotto/scottoslant:default
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottoslant/readme.md:19]:
```suggestion
    make handwired/scottokeebs/scottoslant:default

Flashing example for this keyboard:

    make handwired/scottokeebs/scottoslant:default
```

File: keyboards/handwired/scottokeebs/scottoslant/info.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        "rows": ["B1", "B3", "B2", "B6"]
+    },
+    "url": "https://scottokeebs.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0004",
+        "vid": "0x534B"
+    },
+    "layouts": {
+        "LAYOUT_3x10_7": {
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottoslant/info.json:29]:
```suggestion
        "LAYOUT": {
```

File: keyboards/handwired/scottokeebs/scottoslant/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_3x10_7(
+        KC_Q,         KC_W,   KC_E,    KC_R,    KC_T,           KC_Y, KC_U,          KC_I,          KC_O,   KC_P,
+        KC_A,         KC_S,   KC_D,    KC_F,    KC_G,           KC_H, KC_J,          KC_K,          KC_L,   KC_BSPC,
+        LSFT_T(KC_Z), KC_X,   KC_C,    KC_V,    KC_B,           KC_N, KC_M,          KC_COMMA,      KC_DOT, RSFT_T(KC_SLSH),
+                      KC_ESC, KC_LCTL, KC_LALT, LGUI_T(KC_SPC),       LT(1, KC_TAB), LT(2, KC_ENT), KC_ESC
+    ),
+    [1] = LAYOUT_3x10_7(
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottoslant/keymaps/default/keymap.c:27]:
```suggestion
    [1] = LAYOUT(
```

File: keyboards/handwired/scottokeebs/scottoslant/keymaps/scotto/keymap.c [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+        KC_LSFT, KC_NO,  KC_NO,   KC_NO,   MO(3),          KC_NO,   KC_NO,   KC_COMM, KC_DOT,                    RSFT_T(KC_SLSH),
+                 KC_ESC, KC_LCTL, KC_LALT, LGUI_T(KC_SPC),          KC_TRNS, KC_TRNS, TD(TD_ESC_SPOTLIGHT_EMOJI)
+    ),
+    [3] = LAYOUT_3x10_7(
+        KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,          KC_NO, KC_NO,   KC_NO,   KC_NO,                     TO(4),
+        KC_F1,  KC_F2,  KC_F3,   KC_F4,   KC_F5,          KC_F6, KC_F7,   KC_F8,   KC_F9,                     KC_F10,
+        KC_F11, KC_NO,  KC_NO,   QK_BOOT, KC_TRNS,        KC_NO, KC_NO,   KC_NO,   KC_NO,                     KC_F12,
+                KC_ESC, KC_LCTL, KC_LALT, LGUI_T(KC_SPC),        KC_TRNS, KC_TRNS, TD(TD_ESC_SPOTLIGHT_EMOJI)
+    ),
+    [4] = LAYOUT_3x10_7(
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottoslant/keymaps/scotto/keymap.c:90]:
```suggestion
    [4] = LAYOUT(
```

File: keyboards/handwired/scottokeebs/scottoslant/keymaps/scotto/keymap.c [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+        KC_Q,         KC_W,   KC_F,    KC_P,    KC_G,   KC_J, KC_L,          KC_U,          KC_Y,                    KC_BSPC,
+        KC_A,         KC_R,   KC_S,    KC_T,    KC_D,   KC_H, KC_N,          KC_E,          KC_I,                    KC_O,
+        LSFT_T(KC_Z), KC_X,   KC_C,    KC_V,    KC_B,   KC_K, KC_M,          KC_COMMA,      KC_DOT,                  RSFT_T(KC_SLSH),
+                      KC_ESC, KC_LALT, KC_LCTL, KC_SPC,       LT(5, KC_TAB), LT(6, KC_ENT), TD(TD_ESC_WINDOWS_EMOJI)
+    ),
+    [5] = LAYOUT_3x10_7(
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottoslant/keymaps/scotto/keymap.c:96]:
```suggestion
    [5] = LAYOUT(
```

File: keyboards/handwired/scottokeebs/scottoslant/keymaps/scotto/keymap.c [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+        KC_UNDS,         KC_MINS, KC_PLUS, KC_EQL,  KC_COLN, KC_GRV,   KC_MRWD,       KC_MPLY,       KC_MFFD,                 KC_DEL,
+        KC_LCBR,         KC_LPRN, KC_RPRN, KC_RCBR, KC_PIPE, KC_ESC,   KC_LEFT,       KC_UP,         KC_DOWN,                 KC_RGHT,
+        LSFT_T(KC_LBRC), KC_QUOT, KC_DQUO, KC_RBRC, KC_SCLN, KC_TILDE, KC_VOLD,       KC_MUTE,       KC_VOLU,                 RSFT_T(KC_BSLS),
+                         KC_ESC,  KC_LALT, KC_LCTL, KC_SPC,            LT(5, KC_TAB), LT(6, KC_ENT), TD(TD_ESC_WINDOWS_EMOJI)
+    ),
+    [6] = LAYOUT_3x10_7(
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottoslant/keymaps/scotto/keymap.c:102]:
```suggestion
    [6] = LAYOUT(
```

File: keyboards/handwired/scottokeebs/scottoslant/keymaps/scotto/keymap.c [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+        KC_EXLM, KC_AT,  KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR,       KC_ASTR,       KC_CAPS,                 KC_BSPC,
+        KC_1,    KC_2,   KC_3,    KC_4,    KC_5,    KC_6,    KC_7,          KC_8,          KC_9,                    KC_0,
+        KC_LSFT, KC_NO,  KC_NO,   KC_NO,   MO(7),   KC_NO,   KC_NO,         KC_COMM,       KC_DOT,                  RSFT_T(KC_SLSH),
+                 KC_ESC, KC_LALT, KC_LCTL, KC_SPC,           LT(5, KC_TAB), LT(6, KC_ENT), TD(TD_ESC_WINDOWS_EMOJI)
+    ),
+    [7] = LAYOUT_3x10_7(
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottoslant/keymaps/scotto/keymap.c:108]:
```suggestion
    [7] = LAYOUT(
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Please revert the submodule changes.

**Comment by joe-scotto** [comment]
I believe this is done for all PRs now... please correct me if `git revert` is incorrect.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**